### ISSUE TITLE
安装Esxi Arm Fling时找不到网卡

### ISSUE THREAD

**Issue by fsea** [worproject/rpi5-uefi#20]
Pi5上无法找到网卡
但在Pi4上可以

**Comment by Marcinoo97** [comment]
Yup. Pi 5 has no ethernet support in esxi. VMware never made it. So u just need to use usb ethernet that's compatible.

**Comment by onyxprime-serv** [comment]
This brand worked out of the box for me.
Cable Matters USB to Ethernet Adapter. FBA_202013-BLACK
https://amzn.asia/d/0h1YKLVj
